apps/HelloAndroid/jni/halide.cpp

```diff
@@ -1,4 +1,4 @@
-#include <Halide.h>
+#include "Halide.h"
```

Comment:
That's kinda the end-game (apt-get install libhalide-dev) I could imagine someone downloading a prebuilt, installing it in the usual place, and then copy-pasting code from the tests or building the tutorials.

Anyway, it looks like it works (including "stdio.h" works, at least)